test/e2e/kubectl/portforward.go

```diff
@@ -611,7 +611,10 @@ var _ = SIGDescribe("Kubectl Port forwarding", func() {
 
 			ginkgo.By("Check the client error")
 			gomega.Expect(err).To(gomega.HaveOccurred())
-			gomega.Expect(err.Error()).To(gomega.Or(gomega.ContainSubstring("connection reset by peer"), gomega.ContainSubstring("EOF")))
+			gomega.Expect(err.Error()).To(gomega.Or(
+				gomega.ContainSubstring("connection reset by peer"),
+				gomega.ContainSubstring("EOF"),
+				gomega.ContainSubstring("context deadline exceeded")))
```

Comment:
there is apparently yet another possible error string we could encounter - https://github.com/kubernetes/kubernetes/issues/129527